CourseGrab/Extensions/UserDefaults.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UserDefaults.swift
+//  CourseGrab
+//
+//  Created by Mathew Scullin on 2/9/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+
+extension UserDefaults {
+
+    var get: Bool {
```

Comment:
This is not quite right. Like Kevin said, if we plan to have more user defaults, then we should use [SwiftyUserDefaults](https://github.com/sunshinejr/SwiftyUserDefaults). If not, then this should change to:
```
var didPromptPermission: Bool {
    get {
        return UserDefaults.standard.value(forKey: Constants.UserDefaults.didPromptPermission) as? Bool ?? false
    }
    set(bool) {
        UserDefaults.standard.set(bool, forKey: Constants.UserDefaults.didPromptPermission)
    }
}
```
This has 2 improvements:
 1. More descriptive variable name
 2. You can say stuff like `UserDefaults.standard.didPromptPermission = true`

In fact, I would maybe even hide the constants so that you can't directly access them in anywhere else in the code. i.e.:
```
var didPromptPermission: Bool {
    get {
        return UserDefaults.standard.value(forKey: "didPromptPermission") as? Bool ?? false
    }
    set(bool) {
        UserDefaults.standard.set(bool, forKey: "didPromptPermission")
    }
}
```